pytests/tests/test_datetime.py

```diff
@@ -227,12 +231,11 @@ def test_datetime_typeerror():
 
 
 @given(dt=st.datetimes(MIN_DATETIME, MAX_DATETIME))
-@example(dt=pdt.datetime(1971, 1, 2, 0, 0))
```

Comment:
Why remove this example?

Reply:
It looks like a random case at first. Can add it back in if needed.

Reply:
The datetime tested here is equal to the previous value of `MIN_DATETIME` value on Windows (see line 59 above). I assume that's why it was special cased.